# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1453229180 18000
#      Tue Jan 19 13:46:20 2016 -0500
# Branch stable
# Node ID a58d5e6df908199c4aea83f78077b81289d2a0ad
# Parent  0cfcc178432bfc7cfdef5dca96e74ec5884ba752
maint: Bump version number for 4.0.1-rc3.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -14,24 +14,24 @@ dnl Copyright (C) 1993-2015 John W. Eato
 ### FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
 ### for more details.
 ###
 ### You should have received a copy of the GNU General Public License
 ### along with Octave; see the file COPYING.  If not, see
 ### <http://www.gnu.org/licenses/>.
 
 AC_PREREQ([2.63])
-AC_INIT([GNU Octave], [4.0.1-rc2], [http://octave.org/bugs.html], [octave])
+AC_INIT([GNU Octave], [4.0.1-rc3], [http://octave.org/bugs.html], [octave])
 
 dnl Note that the version number is duplicated here and in AC_INIT
 dnl because AC_INIT requires it to be static, not computed from
 dnl shell variables.
 OCTAVE_MAJOR_VERSION=4
 OCTAVE_MINOR_VERSION=0
-OCTAVE_PATCH_VERSION=1-rc2
+OCTAVE_PATCH_VERSION=1-rc3
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION arg
 OCTAVE_VERSION="$PACKAGE_VERSION"
 
 OCTAVE_COPYRIGHT="Copyright (C) 2015 John W. Eaton and others."
 
 OCTAVE_RELEASE_DATE="2016-01-18"
 
